stubs/setuptools/setuptools/command/bdist_wheel.pyi

```diff
@@ -48,7 +48,7 @@ class bdist_wheel(Command):
     def wheel_dist_name(self) -> str: ...
     def get_tag(self) -> tuple[str, str, str]: ...
     def run(self) -> None: ...
-    def write_wheelfile(self, wheelfile_base: str, generator: str = ...) -> None: ...
+    def write_wheelfile(self, wheelfile_base: str, generator: str = "setuptools (80.9.0)") -> None: ...
```

Comment:
that's going to need to be updated every time we bump the stubtest pin for setuptools, I think; not sure that's worth it